test/extended/cluster/cl.go

```diff
@@ -0,0 +1,147 @@
+package cluster
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	oapi "github.com/openshift/origin/pkg/api"
+	projectapi "github.com/openshift/origin/pkg/project/api"
+	exutil "github.com/openshift/origin/test/extended/util"
+	testutil "github.com/openshift/origin/test/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+)
+
+const deploymentRunTimeout = 5 * time.Minute
+
+var _ = g.Describe("clusterloader", func() {
```

Comment:
How does upstream e2e separate out performance tests?  Add `[Serial][Slow]` to this.

What's end to end time on this?